### PR TITLE
feat: add code review task description

### PR DESCRIPTION
**Pull request by valerydluski**
#### 🤔 This is a ...

- [x] 🌟 New task
- [ ] 🌐 New module
- [ ] ⚙️ Update to an existing task
- [ ] 🔧 Update to an existing module
- [ ] 🔗 Update or addition of external resources or links
- [ ] 🐛 Fix in a task or related content
- [ ] 🛠 Fix in a module or related content
- [ ] ✏️ Fixed a typo or grammatical error
- [ ] 🔗 Fixed a broken link
- [ ] ❓ Other (specify: **\*\*\*\***\_\_\_\_**\*\*\*\***)

#### Description

- **Brief Overview:**
  
- **Implementation Approach:**
  

#### Additional Information

- **Screenshots/Links:**
  
- [ ] Related Issues:

#### Checklist

- [ ] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [ ] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

**Review by stardustmeg — APPROVED** [review]
Everything looks great 👍🏻

**Review by haslie22 — APPROVED** [review]
👍 👍 👍

File: stage2/tasks/code-review/MentorEvaluationCriteria.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+### 1. Identification of Code Issues (Total: 50 points)
+
+- **2 points for each identified issue**.
+- Maximum of 50 points for this criterion.
+- Focus on identifying inefficient code, poor practices, or errors.
+
+### 2. Providing Suggestions (Total: 15 points)
+
+- **3 points for each comment containing suggestions**.
```
Comment (TELEUZI) [inline stage2/tasks/code-review/MentorEvaluationCriteria.md:17]: containing _code improvement_ suggestions

File: stage2/tasks/code-review/MentorEvaluationCriteria.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+This document outlines the evaluation criteria for mentors assessing the code review task. The scoring is based on the student's ability to effectively identify issues, provide suggestions, and maintain professional communication.
+
+## Evaluation Criteria
+
+### 1. Identification of Code Issues (Total: 50 points)
+
+- **2 points for each identified issue**.
+- Maximum of 50 points for this criterion.
+- Focus on identifying inefficient code, poor practices, or errors.
```
Comment (TELEUZI) [inline stage2/tasks/code-review/MentorEvaluationCriteria.md:13]: I think we should add a few examples of poor practices, like:
_Focus on identifying inefficient code, poor practices (code repeat, overcomplicated code, useless code, misleading naming, etc.), or errors._

File: stage2/tasks/code-review/MentorEvaluationCriteria.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+### 2. Providing Suggestions (Total: 15 points)
+
+- **3 points for each comment containing suggestions**.
+- Maximum of 15 points for this criterion.
+- Emphasis on actionable and constructive suggestions.
+
+### 3. Comment Placement Accuracy (10 points)
+
+- **10 points if all comments are accurately placed** relevant to the specific lines of code they address.
```
Comment (TELEUZI) [inline stage2/tasks/code-review/MentorEvaluationCriteria.md:23]: **10 points if all comments are accurately placed**, _meaning they should be directly relevant to the specific lines of code they address._

**Comment by valerydluski** [comment]
@TELEUZI Thank you, I've considered your suggestions

**Pull request merged**